server/Controllers/shiftController/middleware/clockOut.js

```diff
@@ -9,19 +9,17 @@
  * **************************************************
  */
 
-require('dotenv').config();
 const path = require('path');
 const db = require(path.resolve(__dirname, '../../../models/employeeModel'));
 
 const clockOut = async (req, res, next) => {
   const { shift_id } = req.body;
-  let endTime = new Date().toLocaleTimeString();
+  
   const currentTime = Date.now();
   const input = [shift_id, currentTime];
   try {
     let shiftQuery = 
-      ' UPDATE shift SET\
-        end_time = $2 '
+      'UPDATE shift SET end_time = $2 '

```

Comment:
Is the space at the end of this intentional? I'm constantly in aww of how specific these queries need to be

Reply:
Yep - the space at the end is intentional, because if/when we append shiftQuery in line 24, we need there to be a white space in between.